File: src/main/java/no/nav/syfo/controller/domain/sykepengesoknad/oppsummering/Tilleggstekst.java
package no.nav.syfo.controller.domain.sykepengesoknad.oppsummering;

import java.io.Serializable;

public class Tilleggstekst implements Serializable {
    public Ledetekst ledetekst;
    public Svartype type;

    public Tilleggstekst withLedetekst(final Ledetekst ledetekst) {
        this.ledetekst = ledetekst;
        return this;
    }

    public Tilleggstekst withType(final Svartype type) {
        this.type = type;
        return this;
    }
}

File: src/main/kotlin/no/nav/syfo/controller/domain/sykepengesoknad/oppsumering/Tilleggstekst.kt
package no.nav.syfo.controller.domain.sykepengesoknad.oppsumering

import java.io.Serializable

class Tilleggstekst : Serializable {
    var ledetekst: Ledetekst? = null
    var type: Svartype? = null
    fun withLedetekst(ledetekst: Ledetekst?): Tilleggstekst {
        this.ledetekst = ledetekst
        return this
    }

    fun withType(type: Svartype?): Tilleggstekst {
        this.type = type
        return this
    }
}
